src/sandstorm/grain.capnp

```diff
@@ -655,9 +655,9 @@ interface SessionContext {
   # powerboxRequest:
   #   rpcId: A unique string that should identify this rpc message to the app. You will receive this
   #          id in the callback to verify which message it is referring to.
-  #   query: A list of PowerboxDescriptor objects, serialized as a Javascript array OR a
-  #          base64-encoded powerbox query created using the `spk query` tool.
```

Comment:
This is something we plan to add but I suppose we shouldn't have it in the doc if it's not there yet!